stampit.js

```diff
@@ -92,10 +91,7 @@ function compose(factories) {
  * Return a factory function that will produce new objects using the
  * prototypes that are passed in or composed.
  *
- * @param  {Object} [methods] A map of method names and bodies for delegation.
- * @param  {Object} [refs]   A map of property names and values to be mixed into each new object.
- * @param  {Function} [init] A closure (function) used to create private data and privileged methods.
- * @param  {Object} [props]   An object to be deeply cloned into each newly stamped object.
+ * @param  {Object} [options] Options to build stamp from: `{ methods, refs, init, props }`
```

Comment:
There should still be `@param` annotations for each of:

`options.methods`, `options.refs`, `options.init`, and `options.props`.

I wonder if it's possible to define an options type with JSDoc without it assuming that there's an `options` class?

Then you could define the type elsewhere and just do `@param {options} [options] desc...`